Dockerfile

```diff
@@ -50,7 +50,9 @@ RUN npm ci --build-from-source --sqlite=/usr/lib
 
 # We only run these commands when executing through
 # livepush, so they are presented as livepush directives
-#dev-run=apk add --no-cache ip6tables iptables
+# overwrite pre-built node with package version so --inspect can be used
+#dev-run=apk add --no-cache ip6tables iptables nghttp2-dev npm nodejs
```

Comment:
I spoke with Tom and this code is unavoidable because the node version which ships with the Supervisor does not have the inspect components. The issue is that the version installed might differ slightly (patches) then the version being ran outside of livepush. I believe this isn't an issue in reality since test suites do not run with this livepush version so any issues would be found.

Reply:
Is the alpine nodejs package just a different patch version or can it be another version entirely? Even though tests run on the proper environment. If node behavior is different this might reduce usefulness of the inspect option

Reply:
well now I want to see the actual size diff when we compile node with system-icu or small-icu. I haven't been successful testing yet....

Reply:
if we can just ship some ICU to satisfy the interpreter requirements then we won't need this line

Reply:
@pipex there appears to be a single `nodejs` version available for each version of alpine. supervisor is using alpine version 3.11 which has node version 12.22.1-r0. the prebuilt version is 12.16.2.

see https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.11